core/domain/email_manager.py

```diff
@@ -842,7 +831,11 @@ def send_flag_exploration_email(
         exploration_title, report_text, exploration_id,
         EMAIL_FOOTER.value)
 
-    recipient_list = config_domain.MODERATOR_IDS.value
+    recipient_names = user_services.get_usernames_by_role(
+        feconf.ROLE_ID_MODERATOR)
+    recipient_list = []
+    for name in recipient_names:
+        recipient_list.append(user_services.get_user_id_from_username(name))
```

Comment:
rights now, there is no such function for getting all user_ids for a given role.